ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -35,11 +41,11 @@ export class ImportCommand extends IronfishCommand {
 
     let account: AccountImport | null = null
     if (importPath) {
```

Comment:
Instead of using/passing around flags, I think we could just try all the formats (currently hex, base58), and throw useful error if none work. This is better from a usability perspective

Reply:
I see, so for the import codepath, instead of using `--base58`, just try both (for example) `client.importAccount` and `importAccountBase58` and only throw if both fail?